ExchangeRateApp/View/Base.lproj/Main.storyboard

```diff
@@ -40,17 +40,17 @@
                                 <color key="backgroundColor" systemColor="systemBackgroundColor"/>
                                 <prototypes>
                                     <tableViewCell clipsSubviews="YES" contentMode="scaleToFill" preservesSuperviewLayoutMargins="YES" selectionStyle="default" indentationWidth="10" reuseIdentifier="exchangeCell" rowHeight="101" id="gfC-Ta-isZ" customClass="ExchangeTableViewCell" customModule="ExchangeRateApp" customModuleProvider="target">
-                                        <rect key="frame" x="0.0" y="28" width="414" height="101"/>
+                                        <rect key="frame" x="0.0" y="24.5" width="414" height="101"/>
                                         <autoresizingMask key="autoresizingMask"/>
                                         <tableViewCellContentView key="contentView" opaque="NO" clipsSubviews="YES" multipleTouchEnabled="YES" contentMode="center" preservesSuperviewLayoutMargins="YES" insetsLayoutMarginsFromSafeArea="NO" tableViewCell="gfC-Ta-isZ" id="Xm1-P1-cU5">
                                             <rect key="frame" x="0.0" y="0.0" width="414" height="101"/>
                                             <autoresizingMask key="autoresizingMask"/>
                                             <subviews>
-                                                <stackView opaque="NO" contentMode="scaleToFill" spacing="40" translatesAutoresizingMaskIntoConstraints="NO" id="OfL-S0-hEK">
-                                                    <rect key="frame" x="20" y="11" width="373" height="79"/>
+                                                <stackView opaque="NO" contentMode="scaleToFill" distribution="fillEqually" spacing="40" translatesAutoresizingMaskIntoConstraints="NO" id="OfL-S0-hEK">
```

Comment:
Stack views already have constaints between list of elements (AKA arranged subviews), and provide properties to manage spacing between those and their distribution. With stack views you only need to set their edge constraints, elements inside, by default, take up all available space inside.